OpenKeychain/src/main/res/layout/create_key_email_fragment.xml

```diff
@@ -32,6 +32,7 @@
                 android:layout_marginTop="16dp"
                 android:layout_marginBottom="8dp"
                 android:imeOptions="actionNext"
+                android:inputType="textEmailAddress"
```

Comment:
not needed due to https://github.com/Hiperzone/open-keychain/blob/development/OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/widget/EmailEditText.java#L53